ReactCommon/cxxreact/Platform.h

```diff
@@ -18,7 +18,7 @@
 namespace facebook {
 namespace react {
 
-#if (!defined (NOJSC)) && (!V8_ENABLED)
+#if (!_MSC_VER) && (!V8_ENABLED)
 namespace JSCNativeHooks {
```

Comment:
Does anyone need this? This is not needed on droid and windows and I believe Apple is already moved to JSI flow? 

Namrata who is interning in my team had raised this PR where she is getting rid of this block.

[https://github.com/microsoft/react-native/pull/93](https://github.com/microsoft/react-native/pull/93)

Reply:
Do we have any build configuration (OSS/SDX_PLAT/DEVMAIN) which still builds droid with JSC ?

---
In reply to: [297099783](https://github.com/microsoft/react-native/pull/94#discussion_r297099783) [](ancestors = 297099783)